lib/std/builtin.zig

```diff
@@ -443,20 +450,28 @@ pub const Version = struct {
                 if (self.minor == 0) {
                     return std.fmt.format(out_stream, "{}", .{self.major});
                 } else {
-                    return std.fmt.format(out_stream, "{}.{}", .{ self.major, self.minor });
+                    return std.fmt.format(
+                        out_stream,
+                        "{}.{}",
+                        .{ self.major, self.minor },
+                    );
```

Comment:
Hmm, is breaking line here and below really necessary here? FWIW, I think it looked fine the way it was.

Reply:
No, it's not really necessary, but it brought the line under 80 characters which made me happy and it's consistent with the `zig fmt` rules. If there's enough demand, I'll change it back.